docs/workflows.md

```
@@ -0,0 +1,22 @@
+# Github Actions
+We use Github-Actions to build our software. The costs are minimal because we use free and public repositories.
+
+We designed templates to use GitHub Actions. The GitHub Action needs permission on the repository, for example, to create a release or push a tag. Here you can use automatic token authentication from GitHub. Therefore, you can use the access token via ${{ secrets.GITHUB_TOKEN }}. In contrast to the .gitlab-ci.yml, you can create more workflow files which are independent of each other. The it@M templates can be customized to suit your project. You can create reusable actions for single steps.
+
+The templates can be activated under the "Actions" tab with the "New workflow" button. In the software catalog, the templates can be found under the category "By it@m".
+
+
+- Maven-Node-Build: Executes “mvn install” for Maven projects or “npm run build” for Node.js projects. It selects a free version of OpenJDK and Node.js based on the presence of a pom.xml or package.json in the folder. Specify your subfolders if needed. After the source code is built, a “docker build” is executed, and the resulting Docker image is pushed to GitHub’s internal registry with the tag “latest”. Ensure that the Dockerfile is available in the folder.
+-	Maven-Release: This is a manual step. Therefore, you need to select in the tab “actions” on the left the workflow “maven-release” and start it over the button “Run workflow” (on top of the table). After that you can select on the pop-up-menu the version in the format x.y.z  and accordingly SNAPSHOT-X.Y.Z. The manual configuration with write rights is not necessary.
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 2

(MD010, no-hard-tabs)